src/js/transforms.js

```diff
@@ -42,3 +42,47 @@ gpii.binder.transforms.stripEmptyString = function (value) {
 fluid.defaults("gpii.binder.transforms.stripEmptyString", {
     gradeNames: ["fluid.standardTransformFunction"]
 });
+
+/**
+ *
+ * Transform the value returned by jQuery.val for a single checkbox into a boolean.
+ *
+ * @param {Array} value - An array of values, either the value attribute of a ticked checkbox, or "on" if the checkbox has no value specified.
+ * @return {Boolean} - `true` if the first value is checked, `false`.
+ *
+ */
+gpii.binder.transforms.checkToBoolean = function (value) {
+    return fluid.get(value, 0) ? true : false;
+};
+
+gpii.binder.transforms.checkToBoolean.invert = function (transformSpec) {
+    transformSpec.type = "gpii.binder.transforms.booleanToCheck";
```

Comment:
Coverage shows these inversion functions to be untested

Reply:
They are actually exercised by the tests so presumably some failure in the coverage configuration

Reply:
The dev release of gpii-testem addressed this, the coverage is good now.